StripeFinancialConnections/StripeFinancialConnections/Source/Common/ModalPresentationWrapperViewController.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  ModalPresentationWrapperViewController.swift
+//  StripeFinancialConnections
+//
+//  Created by Vardges Avetisyan on 12/22/22.
+//
+
+import UIKit
+
+class ModalPresentationWrapperViewController: UIViewController {
+
+    weak var vc: UIViewController?
+    var observation: NSKeyValueObservation?
+
+    // MARK: - Init
+
+    init(vc: UIViewController) {
+        self.vc = vc
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - UIViewController
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Make wrapper to pass touches through
```

Comment:
yea one scary thing about this "VC hack" is: "What if this view controller, for whatever unknown reason, stays on screen?" 

this likely shouldn't happen (or maybe is completely impossible!), but still a scary scenario; that's why I sometimes add a tap handling on these wrappers so if it somehow stays on screen, tapping it would keep calling "dismiss"

Reply:
ah yeah I had the same exact code for my testing, it's good to have it, I'll bring it back.